XamlControlsGallery/ControlPages/CommandBarFlyoutPage.xaml.cs

```diff
@@ -36,13 +36,13 @@ private void ShowMenu()
         }
 
         private void MyImageButton_ContextRequested(Windows.UI.Xaml.UIElement sender, ContextRequestedEventArgs args)
-        {
-            ShowMenu();
+        {            
+            ShowMenu((sender as Button).IsPointerOver);
```

Comment:
ContextRequested is raised when we right-click to explicitly request a context menu, which should show in standard mode - transient mode is only when the flyout is being shown incidentally in a situation where the user may not want it.